packages/cxl-ui/src/components/cxl-save-playbooks.js

```diff
@@ -0,0 +1,94 @@
+import '@conversionxl/cxl-lumo-styles';
+import { customElement, html, LitElement, property, query } from 'lit-element';
+import cxlSavePlaybooksStyles from '../styles/cxl-save-playbooks-css';
+
+@customElement('cxl-save-playbooks')
+export class CXLSavePlaybooksElement extends LitElement {
+  @query('a')
+  anchor;
+
+  @query('a > iron-icon')
+  ironIcon;
+
+  /**
+   * Is playbook currently in the "saved" state
+   */
+  @property({ type: Boolean, reflect: true })
+  selected;
+
+  /**
+   * if true, text "Save / Unsave Playbook" won't be displayed
+   */
+  @property({ type: Boolean, reflect: true })
```

Comment:
If you don't use the reflected attribute in the CSS, reflecting the attribute is not needed.

Reply:
@anoblet in fact there will be a need to use it in case component will be top-right corner card style.
![cxl-save-playbook-star-only-version](https://user-images.githubusercontent.com/54673942/121856497-d904f780-ccf4-11eb-903c-73f99c96fb51.png)

```
          <cxl-save-playbooks
            playbookId="${playbookId}"
            userId="${userId}"
            ?selected=${playbookSaved}
            ?starOnly=${!saveUnsaveTextVisible}
          ></cxl-save-playbooks>
```

Reply:
If you hide content using a ternary `${!this.starOnly ? html`...` : nothing}` and the attribute is not targeted with CSS (`[starOnly] { display: none; }`) then reflecting is not necessary. You can chose whichever route.

Reply:
agree so reflecting removed on all the properties besides `selected`